src/types.md

```diff
@@ -400,6 +471,33 @@ let bo: Binop = add;
 x = bo(5,7);
 ```
 
+### Other traits
+
+Closure types implement the following traits, if allowed to do so by the
+captured values:
+
+* `[Sized]`
```

Comment:
It feels weird that we're talking about conditionally implemented traits and then the first one on the list is always implemented.

Reply:
Fixed it.